### PR TITLE
Initialize base configuration for releng tenant

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/role-releng-gitops-repo-ro.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (scoheb) [inline cluster/stone-prd-rh01/managed/rhtap-releng-tenant/role-releng-gitops-repo-ro.yaml:1]: tenants-config does not support Roles currently. See https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/stonesoup/argocd-control-plane/rh-managed-workspaces-config-project.yml#L22

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/role-releng-gitops-repo-rw.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (scoheb) [inline cluster/stone-prd-rh01/managed/rhtap-releng-tenant/role-releng-gitops-repo-rw.yaml:1]: tenants-config does not support Roles currently. See https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/stonesoup/argocd-control-plane/rh-managed-workspaces-config-project.yml#L22

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/sa-releng-gitops-repo-ro.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: rhtap-releng-tenant
+  name: releng-gitops-repo-ro
```
Comment (scoheb) [inline cluster/stone-prd-rh01/managed/rhtap-releng-tenant/sa-releng-gitops-repo-ro.yaml:5]: newline

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/sa-releng-gitops-repo-rw.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: rhtap-releng-tenant
+  name: releng-gitops-repo-rw
```
Comment (scoheb) [inline cluster/stone-prd-rh01/managed/rhtap-releng-tenant/sa-releng-gitops-repo-rw.yaml:5]: newline

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/kustomization.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+resources:
```
Comment (scoheb) [inline cluster/stone-prd-rh01/managed/rhtap-releng-tenant/kustomization.yaml:1]: tenants-config currently only supports namespaces named: `rh-managed-*-tenant`

See https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/stonesoup/argocd-control-plane/rh-managed-workspaces-config-project.yml#L18

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/remote-secrets/remote_secret-slack-webhook-notification.yaml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+apiVersion: appstudio.redhat.com/v1beta1
```
Comment (scoheb) [inline cluster/stone-prd-rh01/managed/rhtap-releng-tenant/remote-secrets/remote_secret-slack-webhook-notification.yaml:1]: not needed in this workspace. the pipeline we are targeting does not support the slack task atm. We could add this later.

**Review by scoheb — CHANGES_REQUESTED** [review]
Looks like we have to address some problems with changes to https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/stonesoup/argocd-control-plane/rh-managed-workspaces-config-project.yml first.

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/role-releng-gitops-repo-ro.yaml (hunk 1 above)
Comment (ralphbean) [reply cluster/stone-prd-rh01/managed/rhtap-releng-tenant/role-releng-gitops-repo-ro.yaml]: Dropped in ce39299.

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/role-releng-gitops-repo-rw.yaml (hunk 2 above)
Comment (ralphbean) [reply cluster/stone-prd-rh01/managed/rhtap-releng-tenant/role-releng-gitops-repo-rw.yaml]: Dropped in ce39299.

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/sa-releng-gitops-repo-ro.yaml (hunk 3 above)
Comment (ralphbean) [reply cluster/stone-prd-rh01/managed/rhtap-releng-tenant/sa-releng-gitops-repo-ro.yaml]: Added in 2da7a89.

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/sa-releng-gitops-repo-rw.yaml (hunk 4 above)
Comment (ralphbean) [reply cluster/stone-prd-rh01/managed/rhtap-releng-tenant/sa-releng-gitops-repo-rw.yaml]: Added in 2da7a89.

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/kustomization.yaml (hunk 5 above)
Comment (ralphbean) [reply cluster/stone-prd-rh01/managed/rhtap-releng-tenant/kustomization.yaml:1]: Submitted https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/83373

File: cluster/stone-prd-rh01/managed/rhtap-releng-tenant/remote-secrets/remote_secret-slack-webhook-notification.yaml (hunk 6 above)
Comment (ralphbean) [reply cluster/stone-prd-rh01/managed/rhtap-releng-tenant/remote-secrets/remote_secret-slack-webhook-notification.yaml]: Agreed. Dropped in 094b658.

**Review by scoheb — APPROVED** [review]

**Pull request merged**